### PR TITLE
load_asset_checks_from_package_*

### PR DESCRIPTION
**Pull request by johannkm**
load_asset_checks_from_package_module and load_asset_checks_from_package_name

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #17376** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/17376" target="_blank">Graphite</a> 
    * **PR #17381** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/17381" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/17381?utm_source=stack-comment).

**Review by sryza — APPROVED** [review]

File: python_modules/dagster/dagster/_core/definitions/load_asset_checks_from_modules.py [hunk 1]
```diff
@@ -53,3 +57,85 @@ def load_asset_checks_from_modules(
...
+
+    return _checks_with_attributes(
+        _checks_from_modules([module]), asset_key_prefix=asset_key_prefix
+    )
+
+
+def load_asset_checks_from_package_module(
+    package_module: ModuleType, asset_key_prefix: Optional[CoercibleToAssetKeyPrefix] = None
+) -> Sequence[AssetChecksDefinition]:
+    """Constructs a list of asset checks from all sub-modules of the given package module. This is most often used in conjunction
```
Comment (sryza) [inline python_modules/dagster/dagster/_core/definitions/load_asset_checks_from_modules.py:94]: This looks like it exceeds line length limit

**Review by sryza — APPROVED** [review]
Make sure to check line length limits before merging

File: python_modules/dagster/dagster/_core/definitions/load_asset_checks_from_modules.py (hunk 1 above)
Comment (sryza) [reply python_modules/dagster/dagster/_core/definitions/load_asset_checks_from_modules.py:94]: annoyingly no, just manually

Comment (johannkm) [reply python_modules/dagster/dagster/_core/definitions/load_asset_checks_from_modules.py:94]: we have it set for ruff, but I guess this isn't a hard rule https://github.com/dagster-io/dagster/blob/johann/10-24-asset_checks_batched_loader/pyproject.toml#L170

**Pull request merged**